datafusion/core/tests/fuzz_cases/join_fuzz.rs

```diff
@@ -191,51 +183,47 @@ async fn test_full_join_1k_filtered() {
         JoinType::Full,
         Some(Box::new(col_lt_col_filter)),
     )
-    .run_test(&[JoinTestType::NljHj], false)
+    .run_test(&[NljHj], false)
     .await
 }
 
 #[tokio::test]
-#[allow(unused_qualifications)]
 async fn test_semi_join_1k() {
     JoinFuzzTestCase::new(
         make_staggered_batches(1000),
         make_staggered_batches(1000),
         JoinType::LeftSemi,
         None,
     )
-    .run_test(&[JoinTestType::HjSmj, JoinTestType::NljHj], false)
+    .run_test(&[JoinTestType::HjSmj, NljHj], false)
```

Comment:
Why would we want to import the left but not the right?

Maybe rather than removing the now-redundant `JoinTestType::` qualifications, we could remove `use` statements ?

Reply:
I agree that it is strange for one variant to be qualified and one variant to be unqualified (aka `use`)

Perhaps we could either remove this `use` statement:

```rust
use crate::fuzz_cases::join_fuzz::JoinTestType::NljHj;
```

Or change it to

```rust
use crate::fuzz_cases::join_fuzz::JoinTestType::{HjSmj, NljHj};
```

Reply:
@findepi @alamb Thanks for your review, I agree with you.
I will remove this `use` statement.